src/datasets/vision/imagenet_reader/ImageNetReader.jl

```diff
@@ -0,0 +1,47 @@
+module ImageNetReader
+using ImageCore: channelview, colorview, AbstractRGB, RGB
+
+import ..FileDataset
+import ..read_mat
+import ..@lazy
+
+@lazy import JpegTurbo = "b835a17e-a41a-41e7-81f0-2f016b05efe0"
```

Comment:
This file is called in MLDatasets.jl, are these imports necessary?

Reply:
This is part of the `ImageNetReader` module inside MLDatasets. I thought this was necessary to import packages and functions from the parent module `MLDatasets`.